printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md

```diff
@@ -0,0 +1,8 @@
+# LGX stealthburner connector cover
+
+I was searching for connector cover that could fit my stealthburner LGX toolhead.
+I could not find a cover that would fit my toolhead, so I redesigned [craxoor](https://github.com/craxoor/VoronMods/tree/master/PCB%20Cover) Afterburner PCB Cover. 
+
+![alt text](./Images/Preview-front.png)
```

Comment:
`alt text` is a placeholder.  this should be replaced a textual description of the image.